tests/bot/cogs/test_jams.py

```diff
@@ -0,0 +1,143 @@
+        self.assertTrue(overwrites[leader].connect)
+
+        # Other members permission overwrites
+        for member in members[1:]:
+            self.assertTrue(overwrites[member].read_messages)
+            self.assertTrue(overwrites[member].connect)
+
+        # Everyone and verified role overwrite
+        self.assertFalse(overwrites[self.ctx.guild.default_role].read_messages)
+        self.assertFalse(overwrites[self.ctx.guild.default_role].connect)
+        self.assertFalse(overwrites[self.ctx.guild.get_role(Roles.verified)].read_messages)
+        self.assertFalse(overwrites[self.ctx.guild.get_role(Roles.verified)].connect)
+
+    async def test_team_channels_creation(self):
+        """Should create new voice and text channel for team."""
+        self.utils_mock.get.return_value = "foo"
+        members = [MockMember() for _ in range(5)]
+
+        self.cog.get_overwrites = MagicMock()
+        self.cog.get_category = AsyncMock()
+        self.ctx.guild.create_text_channel.return_value = MockTextChannel(mention="foobar-channel")
+        actual = await self.cog.create_channels(self.ctx, "my-team", members)
+
+        self.assertEqual("foobar-channel", actual)
+        self.cog.get_overwrites.assert_called_once_with(members, self.ctx)
+        self.cog.get_category.assert_awaited_once_with(self.ctx)
+
+        self.ctx.guild.create_text_channel.assert_awaited_once_with(
+            "my-team",
+            overwrites=self.cog.get_overwrites.return_value,
+            category=self.cog.get_category.return_value
+        )
+        self.ctx.guild.create_voice_channel.assert_awaited_once_with(
+            "My Team",
+            overwrites=self.cog.get_overwrites.return_value,
+            category=self.cog.get_category.return_value
+        )
+
+    async def test_jam_roles_adding(self):
+        """Should add team leader role to leader and jam role to every team member."""
+        leader_role = MockRole(name="Team Leader")
+        jam_role = MockRole(name="Jammer")
+        self.ctx.guild.get_role.side_effect = [leader_role, jam_role]
+
+        leader = MockMember()
+        members = [leader] + [MockMember() for _ in range(4)]
+        await self.cog.add_roles(self.ctx, members)
+
+        leader.add_roles.assert_any_await(leader_role)
+        for member in members:
+            member.add_roles.assert_any_await(jam_role)
+
+    async def test_result_sending(self):
+        """Should call `ctx.send` when everything goes right."""
+        members = [MockMember() for _ in range(5)]
+        await self.cog.createteam(self.cog, self.ctx, "foo", members)
+        self.ctx.send.assert_awaited_once()
+        sent_string = self.ctx.send.call_args[0][0]
+
+        self.assertIn(str(self.ctx.guild.create_text_channel.return_value.mention), sent_string)
+        self.assertIn(members[0].mention, sent_string)
+        self.assertIn(" ".join(member.mention for member in members[1:]), sent_string)
```

Comment:
Now that you have things in separate functions, a simpler solution is to assert `create_channels` and `add_roles` were called before `send`. This would turn it into a "general" test for the command rather than being specific to testing if a message was sent, but that's fine.